code/game/dna/dna2_domutcheck.dm

```diff
@@ -3,7 +3,8 @@
 // M: Mob to mess with
 // flags: See below, bitfield.
 /proc/domutcheck(mob/living/M, flags = 0)
-	for(var/datum/mutation/mutation in GLOB.dna_mutations)
+	for(var/mutation_type in GLOB.dna_mutations)
```

Comment:
No clue how I undid this change since I went through all (which weren't that many) instances of `dna_mutations`